example/deploySmartContractLocally.js

```diff
@@ -5,44 +5,45 @@ const solc = require('solc');
 const fs = require('fs');
 
 // Default from address
-const from = '[from address]';
+const from = '';
 
 // EVMLC object
 const evmlc = new EVMLC('127.0.0.1', 8080, {
-    from,
-    gas: 1000000,
-    gasPrice: 0
+  from,
+  gas: 1000000,
+  gasPrice: 0,
 });
 
 // Keystore object
-const dataDirectory = new DataDirectory('[evmlc path]/.evmlc');
-const password = '[password]';
+const dataDirectory = new DataDirectory('');
+const password = '';
 
 
 // Contract Object
-const contractPath = '[contract path]';
+const contractPath = '';
+const rawContractName = '';
+
 const contractFile = fs.readFileSync(contractPath, 'utf8');
-const contractName = ':' + '[contract name]';
+const contractName = ':' + rawContractName;
 
 const output = solc.compile(contractFile, 1);
 const ABI = JSON.parse(output.contracts[contractName].interface);
 const data = output.contracts[contractName].bytecode;
 
 const generateContract = async () => {
+  // Get keystore object from the keystore directory
+  // For the from address so we can decrypt and sign
+  const account = await dataDirectory.keystore.decrypt(from, password);
 
-    // Get keystore object from the keystore directory
-    // For the from address so we can decrypt and sign
-    const account = await dataDirectory.keystore.decrypt(from, password);
+  // Generate contract object with ABI and data
+  const contract = await evmlc.generateContractFromABI(ABI, data);
 
-    // Generate contract object with ABI and data
-    const contract = await evmlc.generateContractFromABI(ABI, data);
+  // Deploy and return contract with functions populated
+  const response = await contract.deploy(account, { parameters: [100000] });
```

Comment:
What is the `parameters` value used for? 🤔

Reply:
The explanation is required. @mushroom0117, please clarify.

Reply:
It is meant to be the constructor parameters of the Contract.

Reply:
Insert some relevant comment?

Reply:
In the before example code, we use CrowdFunding contract for example.
so the [100000] is a list of the parameters, takes in a value for the limit to the total contribution.
But we have remove the hardcode, so this should be remove, too.

Reply:
Roger. Refactor the sample code.